org.eclipse.winery.repository.rest/src/test/resources/entitytypes/interfaces/add_input_params.json

```diff
@@ -0,0 +1,29 @@
+[
+    {
+        "operation": [
+            {
+                "inputParameters": {
+                    "inputParameter": [
+                        {
+                            "name": "Param1",
+                            "type": "xsd:string",
+                            "required": "YES"
+                        },
+                        {
+                            "name": "Param2",
+                            "type": "xsd:string",
+                            "required": "YES"
```

Comment:
I would update one of the params to be not required. Then we have both variants tested.